vpr/src/route/rr_graph_reader.cpp

```diff
@@ -733,13 +739,30 @@ void process_rr_node_indices(const DeviceGrid& grid) {
         } else if (node.type() == CHANX) {
             for (int ix = node.xlow(); ix <= node.xhigh(); ix++) {
                 for (int iy = node.ylow(); iy <= node.yhigh(); iy++) {
-                    indices[CHANX][iy][ix][0].push_back(inode);
+                    coordinates_max_ptc[iy][ix] = std::max(coordinates_max_ptc[iy][ix], node.ptc_num());
```

Comment:
We could waste some memory by allocating these vectors to store the max of the ptc_num in the CHANX or CHANY direction for both channels. Some FPGAs have different channels widths in the x- and y-directions, so these may be significantly different. I suggest you compute two different coordinates_max_ptc values for CHANX and CHANY at each location, and use them to size the vectors for the appropriate indices.

Reply:
Makes sense, done.